tests/test_pro/test_pro_dpm.py

```diff
@@ -0,0 +1,60 @@
+import os
+
+from util.fixture_fluent import download_input_file
+
+
+@pytest.mark.solve
```

Comment:
@akulshre-qa missing pytest import

Reply:
Thanks Sean. Just pushed the updated one.